app/components/SearchableMetricSelector.js

```diff
@@ -0,0 +1,61 @@
+import { useEffect, useState } from 'react'
+
+function Modal(props){
+  const { supportedMetrics, onClose, selectMetric, show } = props;
+  if(!show){
+    return null;
+  }
+  
+  return (
+    <div className="SelectorModal" onClick={onClose} >
+        <div className="modal-content" onClick={e => e.stopPropagation() } >
+          <div className="modal-header" >
+            <h4 className="modal-title">Select your Metric</h4>
+          </div>
+          <div className="modal-body" >
+            <span>
+              {Object.keys(supportedMetrics).map((k, i) => (
+                <p 
```

Comment:
This is good for now, later we can expand this to be more than just a paragraph.